changelog/6387.feature.rst

```diff
@@ -0,0 +1 @@
+Enable the usage of super().from_parent for Node subclasses.
```

Comment:
Also not sure this is needed.

Reply:
The `_create` meta class method is no public api

Reply:
Yes, but perhaps this should be mentioned in the original news item instead?

Reply:
its different items, while one os a collateral of the other i still think 2 entries are warranted and necessary